Automattic/sensei
# Fix preview styles for Course Outline block on the editor

Resolves #7305

## Proposed Changes

* Tweak the course patterns list on the Course wizard.
  * Fix Course Outline styles on preview thumbnails.
  * Align take course button in the pattern preview to be left-aligned.
  * Tweak pattern spacings around the thumbnails.
  * Update Course Progress to say "0 of 3 lessons completed (0%)" on the preview.
  * Update Course Outline preview to not have modules.

~~Pending questions: https://github.com/Automattic/sensei/issues/7305#issuecomment-1821490797~~

## Testing Instructions


1. Create a new course.
2. Navigate to the patterns list, and check that the styles look better now, reflecting all the proposed changes.
3. Check that the patterns list on Lesson wizard didn't have any bad impact.

## Screenshots

Course patterns before

<img width="830" alt="Screenshot 2023-11-21 at 16 05 18" src="https://github.com/Automattic/sensei/assets/876340/6b74949e-0e3a-4a4c-b8f0-d51f6e16afb5">

Course patterns after

<img width="813" alt="Screenshot 2023-11-22 at 15 01 22" src="https://github.com/Automattic/sensei/assets/876340/6ff49e85-bb5d-4fa6-8b8f-23748f030126">

Lesson patterns before

<img width="820" alt="Screenshot 2023-11-21 at 16 06 17" src="https://github.com/Automattic/sensei/assets/876340/7b67b273-5684-4eb0-9f0b-ea16b2124448">

Lesson patterns after

<img width="835" alt="Screenshot 2023-11-21 at 16 06 38" src="https://github.com/Automattic/sensei/assets/876340/c76887e3-e38e-42ec-97bf-4bfe39e1be6c">

## Known issues

* https://github.com/Automattic/sensei/pull/7307/files#r1401092030
* https://github.com/Automattic/sensei/issues/7312

## Pre-Merge Checklist
